apps/studio/components/interfaces/Settings/Addons/Addons.tsx

```diff
@@ -249,14 +250,15 @@ const Addons = () => {
                       </p>
                     )}
                     <ProjectUpdateDisabledTooltip
-                      projectUpdateDisabled={projectUpdateDisabled}
+                      projectUpdateDisabled={computeSizeChangesDisabled}
```

Comment:
We don't want to disable this when `projectUpdateDisabled` is enabled as well?
```suggestion
                      projectUpdateDisabled={computeSizeChangesDisabled || projectUpdateDisabled}
```

Reply:
so disabling compute sizes will be its own flag, separate from the flag that controls "project creation and update"

Reply:
hmm actually good point...

Reply:
yup okay i'll add that in! thanks man! 🙏 🙏